### ISSUE TITLE
Error: Type "Json" was defined more than once

### ISSUE THREAD

**Issue by Fi1osof** [Urigo/merge-graphql-schemas#158]
Bug.

```
let user = `

  scalar Json

  type User {
    id: ID!
    data: Json 
  } 
`
```
...
```
let resource = `

  scalar Json

  type Resource {
    id: ID!
    data: Json 
  } 
`
mergeTypes([user, resource], { all: true });

Error: Type "Json" was defined more than once

**Comment by iamrommel** [comment]
i created a pull request for this but it was not yet merged on the main branch because of lacking test, which i was not able to create because i'm away from holiday

https://github.com/okgrow/merge-graphql-schemas/pull/157

**Comment by otbe** [comment]
This bites me too, but for `enum` definitions. 
Any chance to fix this too? :)

**Comment by kamilkisiela** [comment]
Should be [fixed](https://github.com/Urigo/merge-graphql-schemas/pull/196/files#diff-ee59d0638e75272d8511d2bc3e6ae758R809) by #196

**Comment by kamilkisiela** [comment]
It's available now under `v1.6.0-beta.2`. Let us know if it works :)
